app/models/article.js

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
Yea I share that opinion too, but I am thinking; wouldn't we be destroying an already existing plan to add a new feature, since we are working with TDD, the test already existing means there were probably plans on ground for something, I think maybe it will be better if we try to figure out the plan before completely removing the file.